Code/Framework/AzToolsFramework/AzToolsFramework/UI/PropertyEditor/EntityPropertyEditor.cpp

```diff
@@ -742,6 +742,22 @@ namespace AzToolsFramework
         m_newComponentId = componentId;
     }
 
+    void EntityPropertyEditor::GetComponentEditors(const GetComponentEditorsCallback& callback) const
+    {
+        for (const ComponentEditor* componentEditor : m_componentEditors)
+        {
+            if (!componentEditor)
+            {
+                continue;
+            }
+
+            if (!callback(componentEditor))
+            {
+                return;
```

Comment:
question: Just was curious what this early out was for, is this so all further processing of component editors can stop? What might this be useful? Also is it worth returning this value to the outer function? Just curious if the result might be useful to know/detect.

Reply:
1. Yes, it's basically an early-out mechanism. If the caller wants to know if atleast one component editor of a type exists or if a component editor matching a particular id/name exists, they can return false to stop iterating over the rest of the components. 
2. I don't think this function should be returning the value because the idea of implementing this with a function is so that api doesn't directly copy or return the component editors. Besides, the callback will have access to the editors. So callers will still be able to know when the stoppage point happens.

Reply:
Cool makes sense, thanks for the explanation 👍